src/component/TimeLayerSliderPanel/TimeLayerSliderPanel.tsx

```diff
@@ -245,10 +349,10 @@ export class TimeLayerSliderPanel extends React.Component<TimeLayerSliderPanelPr
   */
   onTimeChanged(val: string) {
     this.setState({
-      value: moment(val)
+      value: moment(val).clone()
```

Comment:
Shouldn't `moment(val)` be a new object anyway?

Reply:
You're right of course.. Removed clone call in 1cf43a5